src/routes/auth/refresh/server.test.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, it, expect, vi } from 'vitest';
+import { COOKIE_SESSION_NAME } from '../constants';
+import { POST } from './+server';
+
+vi.mock('@sveltejs/kit', () => ({
+  json: vi.fn((data) => data),
+}));
+
+describe('POST function', () => {
+  it('should set the cookie if token is provided', async () => {
+    const mE = {
```

Comment:
Change the name of the variable.
```suggestion
    const mockEvent = {
```